prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -56,11 +57,25 @@ public class EnrolleesController : PrimeControllerBase
         [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<EnrolleeListViewModel>>), StatusCodes.Status200OK)]
         public async Task<ActionResult> GetEnrollees([FromQuery] EnrolleeSearchOptions searchOptions)
         {
+            Console.WriteLine("---- entered controller");
+
             if (User.IsAdministrant())
             {
+                var sw = Stopwatch.StartNew();
                 var notifiedIds = await _enrolleeService.GetNotifiedEnrolleeIdsForAdminAsync(User);
+                sw.Stop();
+                System.Console.WriteLine($"---- first db call: {sw.ElapsedMilliseconds}ms");
+
+                sw = Stopwatch.StartNew();
                 var enrollees = await _enrolleeService.GetEnrolleesAsync(searchOptions);
+                sw.Stop();
+                System.Console.WriteLine($"---- second db call: {sw.ElapsedMilliseconds}ms");
+
+                sw = Stopwatch.StartNew();
                 var result = enrollees.Select(e => e.SetNotification(notifiedIds.Contains(e.Id)));
+                sw.Stop();
```

Comment:
If the `notifiedIds` gets too large would the `Contains(e.Id)` also cause a delay which is not be related to DB performance? I noticed that the last `WriteLine` is not writing about DB call round time measurement but just wanted to confirm that's not what we're looking for in the last logged message.